partner_capital/views/res_partner_turnover_range_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Antiun Ingeniería S.L. - Antonio Espinosa
```

Comment:
If AGPL license must be applied on code, I do not care about the language used. Odoo views contain logic written in XML, and I consider it should be copyrighted too. I know it's not required in OCA, but nevermind.